src/services/restaurants/restaurants.go

```diff
@@ -0,0 +1,116 @@
+package restaurants
+
+import (
+	"github.com/Nastya-Kruglikova/cool_tasks/src/models"
+	"github.com/Nastya-Kruglikova/cool_tasks/src/services/common"
+	"github.com/gorilla/mux"
+	"github.com/satori/go.uuid"
+	"net/http"
+)
+
+type successAdd struct {
+	Status string `json:"message"`
+}
+
+type successDelete struct {
+	Status string `json:"message"`
+}
+
+//Get used for getting restaurants
+func Get(w http.ResponseWriter, r *http.Request) {
```

Comment:
Could you write methods Get, SaveRestaurant, Delete, GetRestaurantFromTrip in one style ?